### PR TITLE
Add message if space already exists, re-enable button

### PR DESCRIPTION
**Pull request by wow-such-code**

### CODE REVIEW

File: src/main/java/life/qbic/projectwizard/views/AdminView.java [hunk 1]
```diff
@@ -167,8 +167,13 @@ public void buttonClick(ClickEvent event) {
                   "There seems to have been a problem while creating the space. Do the specified users already exist in openbis? If not, create them.",
                   NotificationType.ERROR);
             }
-            createSpace.setEnabled(true);
+
+          } else {
+            Styles.notification("Problem creating space",
+                    "This space already exists.",
+                    NotificationType.DEFAULT);
```
Comment (Steffengreiner) [inline src/main/java/life/qbic/projectwizard/views/AdminView.java:174]: NIT: you could also add the space name in this notification

**Review by Steffengreiner — APPROVED** [review]
Nicely done 🚀

File: src/main/java/life/qbic/projectwizard/views/AdminView.java (hunk 1 above)
Comment (wow-such-code) [reply src/main/java/life/qbic/projectwizard/views/AdminView.java:174]: It's still shown in the UI in this case, so not necessary.

Comment (Steffengreiner) [reply src/main/java/life/qbic/projectwizard/views/AdminView.java:174]: Perfect 👍

**Pull request merged**